go_node_engine/model/Service.go

```diff
@@ -18,6 +18,7 @@ type Service struct {
 	UnikernelImages []string `json:"vm_images"`
 	Architectures   []string `json:"arch"`
 	Pid             int
+	OneShot			bool	 `json:"one_shot"`
```

Comment:
can we fix alignments? Is it tab indented or space or mixed?

Reply:
Nicely spotted.
Strange, on my end this looks like this:
![image](https://github.com/oakestra/oakestra/assets/65814168/c15f4be0-335e-4814-89a2-a2fd291c568d)
I guess github did its thing

Is fixed